pkg/operation/botanist/worker.go

```diff
@@ -38,7 +36,7 @@ import (
 
 // WorkerDefaultTimeout is the default timeout and defines how long Gardener should wait
 // for a successful reconciliation of a worker resource.
-const WorkerDefaultTimeout = 30 * time.Minute
+const WorkerDefaultTimeout = 5 * time.Minute
```

Comment:
Is it a reasonable timeout? Just had a worker on aws with machdeploy with 2 replicas that took ~8m to be deleted.

Reply:
The motivation was to display potential errors faster, that's why I chose a rather short time. If it doesn't complete within that time it will be retried anyways by the shoot controller. However, do you think it makes sense to set it to `10m`?

Reply:
I would rather pick `10m` to make sure that the worker succeeds. My idea is to prevent timeout/retry at this step to be something constant for shoot with several general purpose machines. 
The mcm guys can also share their experience across different providers. If you previously agreed on `5m`, I am also fine.

Reply:
Agreed, changed to `10m` with 2c56d9121f0425269f507515cf8181ad25f876f3.